.github/workflows/no-response.yml

```
@@ -18,13 +18,15 @@ jobs:
       contents: read
       issues: write
     steps:
-      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb # v0.5.0
+      - uses: MBilalShafi/no-response-add-label@9657b5bc1e09373e571df54429c3c167edaad556
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           # Number of days of inactivity before an Issue is closed for lack of response
           daysUntilClose: 7
           # Label requiring a response
-          responseRequiredLabel: 'status: waiting for follow-up'
+          responseRequiredLabel: 'status: waiting for author'
+          # Label to add back when required label is removed
```

I got confused by this comment. Would it be clearer like this?

```suggestion
          # The label that will be added back when the `responseRequiredLabel` is removed due to the author responding back.
```